### PR TITLE
Improves presentation of tied highest/lowest rates

### PR DESCRIPTION
**Pull request by benhammondmusic**
## Preview Link

<a href="https://deploy-preview-1941--health-equity-tracker.netlify.app/exploredata?mls=1.women_in_legislative_office-3.00&dt1=women_us_congress#rate-map" target="_blank">
  <br />
Click to demo updates </a> 

## Description

- refactors as re-useable component and logic
- changes presentation to one-line tied rates
- prefer `boolean` to `Boolean`, was causing weirdness
- 

## Motivation and Context

#1939 

## How has this been tested?

tests still passing

## Screenshots (if appropriate):

Mixed, Normal, Both Tied
Screen Shot 2022-12-16 at 12 18 19 PM
Screen Shot 2022-12-16 at 12 18 40 PM
Screen Shot 2022-12-16 at 12 18 59 PM

## Types of changes

- New feature (non-breaking change which adds functionality or updates content)
- Refactor (code improvement that doesn't affect UI)

### CODE REVIEW

File: frontend/src/cards/ui/HighestLowestList.tsx [hunk 1]
```diff
@@ -79,58 +77,21 @@ export function HighestLowestList(props: HighestLowestListProps) {
...
-                    if (props.qualifierItems?.includes(placeName)) {
-                      placeName += ` ${props.qualifierMessage}`;
-                    }
+              <ExtremeList
+                whichExtreme="Highest"
+                values={props.highestValues}
+                metricConfig={props.metricConfig}
+                qualifierMessage={props.qualifierMessage}
+                qualifierItems={props.qualifierItems}
+              />
```
Comment (eriwarr) [inline frontend/src/cards/ui/HighestLowestList.tsx:86]: Wow, so much easier to read now.

File: frontend/src/cards/ui/UnknownsAlert.tsx [hunk 2]
```diff
@@ -23,11 +23,11 @@ interface UnknownsAlertProps {
...
-  known: Boolean;
-  overrideAndWithOr?: Boolean;
-  raceEthDiffMap?: Boolean;
-  noDemographicInfoMap?: Boolean;
-  showingVisualization?: Boolean;
+  known: boolean;
+  overrideAndWithOr?: boolean;
+  raceEthDiffMap?: boolean;
+  noDemographicInfoMap?: boolean;
+  showingVisualization?: boolean;
```
Comment (eriwarr) [inline frontend/src/cards/ui/UnknownsAlert.tsx:30]: Great catch.

**Review by eriwarr — APPROVED** [review]
Great refactor!

File: frontend/src/cards/ui/UnknownsAlert.tsx (hunk 2 above)
Comment (benhammondmusic) [reply frontend/src/cards/ui/UnknownsAlert.tsx:30]: only caught it because i mixed them up originally myself

**Review by joshzarrabi — APPROVED** [review]

**Pull request merged**